cryptotaxcalculator/axios-cached-dns-resolve
# 3.3.2: CJS module + Typescript/Type safety + Jest + CI Pipeline

## Description

The `axios-cached-dns-resolve` package is an ESM package that doesn't include types.

This PR:
- Changes the package to be CJS 
- Uses Typescript (for type safety)
- Uses Jest as the test runner
- Adds GitHub Actions CI pipeline

## Testing

1. Run `npm install` to install dependencies
2. Run `npm run build` to verify that it builds
3. Run `npm test` to verify that tests pass
